samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -131,6 +135,66 @@ static void print_qos(struct bt_codec_qos *qos)
 	       qos->rtn, qos->latency, qos->pd);
 }
 
+/**
+ * @brief Send audio data on timeout
+ *
+ * This will send an increasing amount of audio data, starting from 1 octet.
+ * The data is just mock data, and does not actually represent any audio.
+ *
+ * First iteration : 0x00
+ * Second iteration: 0x00 0x01
+ * Third iteration : 0x00 0x01 0x02
+ *
+ * And so on, until it wraps around the configured MTU (CONFIG_BT_ISO_TX_MTU)
+ *
+ * @param work Pointer to the work structure
+ */
+static void audio_timer_timeout(struct k_work *work)
```

Comment:
Do you plan to add LC3 encoder as well?

Reply:
That is probably a good idea, but the change in this sample was mostly to be able to test the stack change. The LC3 support should be added in a separate PR: https://github.com/zephyrproject-rtos/zephyr/issues/45595